presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -90,6 +90,29 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table Properties
```

Comment:
@Praveen2112 thoughts providing some SQLServer reference here that explains the properties in more detail?

Reply:
well... is any property supported? If yes.. we should list them all or link to documentation where they are listed.

Reply:
As of now we support only one property.